sdk/typescript/tsconfig.json

```diff
@@ -5,7 +5,8 @@
 		"module": "CommonJS",
 		"outDir": "dist/cjs",
 		"rootDir": "src",
-		"isolatedModules": true
+		"isolatedModules": true,
+		"sourceMap": true
```

Comment:
Needed for debugging TS code in the IDE

Reply:
It's impossible to comment on this file.